Appraisals

```diff
@@ -0,0 +1,7 @@
+appraise "rails-4" do
+  gem "rails", "4.2.7"
```

Comment:
If we want to make sure that the gem works fine with the latest Rails security patches, we should use those versions. I propose using something like ~> 4.2.7, though.